config/resources/users-domain.lisp

```diff
@@ -16,7 +16,9 @@
   :class (s-prefix "foaf:OnlineAccount")
   :resource-base (s-url "http://kanselarij.vo.data.gift/id/accounts/")
   :properties `((:provider    :via ,(s-prefix "foaf:accountServiceHomepage"))
-                (:vo-id       :via ,(s-prefix "dct:identifier")))
+                (:vo-id       :via ,(s-prefix "dct:identifier"))
+                (:vo-email       :via ,(s-prefix "ext:email"))
+                (:phone-number       :via ,(s-prefix "ext:phone")))
   :has-one `((user            :via ,(s-prefix "foaf:account")
```

Comment:
If you have a lisp file where the indentations don't match up, increase the other indentations to keep it clean.

Reply:
![image](https://user-images.githubusercontent.com/11557630/76784295-a7c90f00-67b3-11ea-9ed8-ebbed7a2d8bd.png)
This was the best i could do